pages/linux/csplit.md

```diff
@@ -0,0 +1,24 @@
+# csplit
+
+> Split a file into pieces.
+> This generates files named "xx00", "xx01", and so on.
+
+- Split a file at lines {{5}} and {{23}}:
```

Comment:
5 and 23 need not be inside `{{}}`. We use that only for examples.

Reply:
Removed from this line and the other example descriptions in fe8cd972